pvr.vuplus/changelog.txt

```diff
@@ -1,3 +1,10 @@
+v3.7.4
+- Fixed: timeshift support for Live TV in v18 
```

Comment:
This not a fix, but a new feature. I would phrase it simply "Added: timeshift support"